### PR TITLE
create router - connect router to home page

### PR DESCRIPTION
**Pull request by ghost**
relates #28

### CODE REVIEW

**Review by KatiaAsh — APPROVED** [review]

**Review by veredrec — APPROVED** [review]

**Review by Heathercoraje — APPROVED** [review]

**Review by Heathercoraje — APPROVED** [review]

**Review by Heathercoraje — APPROVED** [review]

**Comment by Heathercoraje** [comment]
Lawson, I approved then github kept removing me from reviewers. lol

File: src/controllers/route.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const path = require('path');
+const router = express.Router();
+
+const home = require('../views/home');
+
+router.get('/', (req, res) => {
+   res.render('home', { activePage: {home: true}});
+});
+
```
Comment (philawsophizing) [inline src/controllers/route.js:10]: Why are you sending data on both routes (activePage: {home: true}) ? Remember the second argument is where you send data for handlebars to use. This active page data was just used in the code along to set a class which underlined the link.

You can omit this for now, just use res.render('home');

File: src/views/layouts/main.hbs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+{{>htmlHead}}
+   <body>
+      {{>header}}
+      
+      <main class="main">
+         {{{body}}}
+      </main>
+      {{>footer}}
```
Comment (philawsophizing) [inline src/views/layouts/main.hbs]: Here for the footer, only include a partial if it exists.

**Review by philawsophizing — CHANGES_REQUESTED** [review]
Quick changes, change them on the same branch locally then re-push that branch :maple_leaf:

File: src/controllers/route.js (hunk 1 above)
Comment (Heathercoraje) [reply src/controllers/route.js:10]: Thanks for the explanation. We certainly didn't have an idea what the second argument does.
We've fixed it.

Comment (Heathercoraje) [reply src/controllers/route.js:10]: We've added {{footer}} as our very next step is to build header and footer. Our team has agreed on leaving a indicator for immediate codes to be built. Though we will make sure we don't get ahead of ourselves too much. Thank you Lawson.

Comment (philawsophizing) [reply src/controllers/route.js:10]: I figured that, so deleted that footer comment, but apparently not fast enough :smile: 

Is the fix for res.render being pushed to the 'paul2' branch so I can merge it?

Comment (veredrec) [reply src/controllers/route.js:10]: We pushed it now with other changes which we worked on since then :nerd_face: 
It's #30
Thanks!

Comment (Heathercoraje) [reply src/controllers/route.js:10]: We were adding codes in another branch already but we've added a fix to the branch called 'headerPartial'

**Pull request merged**